ding/league/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .player import Player, ActivePlayer, HistoricalPlayer, create_player
 from .starcraft_player import MainPlayer, MainExploiter, LeagueExploiter
 from .shared_payoff import create_payoff
+from .metric import get_elo, get_elo_array
```

Comment:
> naive trueskill demo
> naive ELO demo
> SC2 MMR demo

Is it possible we could move SC2 MMR demo into another PR? Training on full SC2 environment and making MMR increase to a very high level might take weeks/months to finish. Let's close issue #18 first. Please correct me if I am wrong about the training time.

Reply:
It indeed take weeks to train SC2 agent and receive a reasonable result. But in this PR, I just add the MMR calculation algorithm applied in our DI-star project.